HipercowApi/Models/LdapConnectionWrapper.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Imperial College London. All rights reserved.
+
+using System.DirectoryServices.Protocols;
+using Microsoft.AspNetCore.Mvc;
+
+/// <summary>
+/// A simple wrapper over an LdapConnection,
+/// or the error returned trying to make one.
+/// </summary>
+public class LdapConnectionWrapper
```

Comment:
I don't think you should need this wrapper if you use HttpResponseExceptions to handle errors instead.

Reply:
The wrapper is now gone...